test/manipulation.test.js

```diff
@@ -901,6 +902,103 @@ describe('manipulation', function () {
         e.should.be.eql(new Error('Invalid date: X'));
       }
     });
+  });
+
+  describe('update|updateAll', function() {
+    beforeEach(function destroyFixtures(done) {
+      Person.destroyAll(done);
+    });
+
+    beforeEach(function createFixtures(done) {
+      var people = [{
+        name: 'foo',
+        age: 19
+      }, {
+        name: 'bar',
+        age: 20
+      }, {
+        name: 'baz',
+        age: 21
+      }, {
+        name: 'qux',
+        age: 22
+      }, {
+        name: 'quux',
+        age: 23
+      }];
```

Comment:
Sure. Out of curiousity though, are real names better than usual programming placeholder names?